app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+        self.wife = None
+        self.husband = None
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person_data in people:
+        person = Person(person_data["name"], person_data["age"])
+        person_list.append(person)
+
+    for ind, person_data in enumerate(people):
+        if "wife" in person_data and person_data["wife"] is not None:
+            person_list[ind].wife = [person for person in person_list
+                                     if person.name == person_data["wife"]][0]
```

Comment:
There is no need to use `for` and `if`; this can be done more simply
```suggestion
            person_list[ind].wife = Person.people[person["wife"]]
```